k8s/helm/yona/templates/10_deployment_admin.yaml

```diff
@@ -49,6 +64,14 @@ spec:
           volumeMounts:
             - name: config-volume
               mountPath: /opt/app/config
+          livenessProbe:
+            httpGet:
+              path: /health/
+              port: 9080
+            initialDelaySeconds: 60
+            periodSeconds: 15
+            timeoutSeconds: 3
+            failureThreshold: 3
```

Comment:
This implies the service needs to start in 105 seconds, right? 60 + 15 * 3. Isn't that a but a narrow window? Or am I misunderstanding it?

Reply:
The admin container will not be checked at all for 60 seconds (enough time for springboot to turn up and get the /health endpoint working in my tests).   
After that, the K8S will check /health every 15 seconds, and if it does not respond in 3 seconds, 3 times in a row, K8S will shoot the instance in the head, and start a new one.

Reply:
We might need to tune these further.